dev/tasks/homebrew-formulae/autobrew/apache-arrow.rb

```diff
@@ -25,8 +25,8 @@ class ApacheArrow < Formula
 
   bottle do
     cellar :any
-    sha256 "b00e518cda38f90ecefaf542d4c030a966eae50623b057e9edb6fbe93fc3f4ab" => :el_capitan_or_later
-    root_url "https://jeroen.github.io/bottles"
+    sha256 "a55211ba6f464681b7ca1b48defdad9cfbe1cf6fad8ff9ec875dc5a3c8f3c5ed" => :el_capitan_or_later
+    root_url "https://autobrew.github.io/bottles"
   end
```

Comment:
Is this something that needs to be automated in the next release?

Reply:
No, can't be automated, it's part of a post-release task that Jeroen does to build the autobrew bottle. The change isn't really relevant for us anyway, I just updated it since I was updating other things in the script to keep in sync with upstream.